libs/@local/hash-backend-utils/src/queue/redis.ts

```diff
@@ -178,9 +171,9 @@ export class RedisQueueExclusiveConsumer implements QueueExclusiveConsumer {
     let item = await this.client.rpoplpush(processingName, processingName);
```

Comment:
Can't comment this far from the diff but there's a typo in the doc-comment above

 - timemoutMs === 0: blocks indefinitely until an item appears on the queue.

should be `timeoutMs`

Reply:
Thanks 31b2244fe3b18daf130d84f1c3e958304674b3ff